src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +11,15 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        String user1email = "bob@i.ua";
+        String passworduser1 = "1234";
+        String user2email = "alice@i.ua";
+        String passworduser2 = "1234";
```

Comment:
you don't need to hardcode these values
you should create `new UserService()` object and call its method to find User by email
and then compare his password with given password